### ISSUE TITLE
Issues on MacOs

### ISSUE THREAD

**Issue by Eversor** [Eisenhuth/unity-appgroups#1]
Hello, i was trying to use the code provided but it doesn't seem to work if i follow the guide. First i had unity complaining about some files in the plugin not having a meta file. Then when i followed a guide to move the plugin from the Library/PackageCache to the Packages folder i still had issues with the namespace being off compared to the description. I still didn't manage to get it to work unfortunately, but thank you for sharing.

**Comment by Eisenhuth** [comment]
So there are a few parts to this, first and foremost the package is designed for iOS which is what I used and built it for, however there's a good chance you can get away with changing the `#if UNITY_IOS` to be `#if UNITY_IOS || UNITY_STANDALONE_OSX`

I can't really provide support for moving the package/"installing" it by moving it to some folder. Unity is very picky about how the `Plugins/iOS` directories work/are accepted

Lastly, after the whole fiasco surrounding Unity recently I personally have no real desire to work on/with it outside of work for now, but I'll add it to the list

**Comment by Eversor** [comment]
Sorry, from your reply i see that i wasn't completely clear about it. I installed the package on a mac to build for IOS, and it was complaining about the meta missing for the LICENSE and the README.md files. Not sure if they can be excluded from the plugin automatically or a meta file should be provided instead.

The way i found to fix the issue is moving the packages from the Library/PackageCache to the Packages folder as suggested on a forum post.

I also had to change the call from 
var appGroupPath = Plugins.iOS.AppGroupPlugin.GetAppGroupPath("YOUR.GROUP.IDENTIFIER");
to
var appGroupPath = AppGroupPlugin.Plugins.iOS.AppGroupPlugin.GetAppGroupPath("YOUR.GROUP.IDENTIFIER");

I'm not criticizing in any way, you really saved me days of work. I'd just like to help other people that might end up with the same issue.
